### ISSUE TITLE
Maybe improve the toggles here

### ISSUE THREAD

**Issue by NS21fpsy** [humdek-unibe-ch/sh-selfhelp#508]
![image](https://github.com/user-attachments/assets/5f355f76-3930-491d-bdfb-337462bbe1d0)

Hey, the option to delete columns is great.

It might be a bit difficult to make sure that we only select the columns we want when the toggles are so far away from the column names.

What do you think? could you make it clearer which toggle goes with which column name?

thanks

**Comment by Kodzhabashev** [comment]
Good point, I will add some border and maybe transform the toggle to checkbox and move it in front of the column name.
